lib/src/sql/statement.rs

```diff
@@ -202,27 +225,36 @@ pub fn statement(i: &str) -> IResult<&str, Statement> {
 	delimited(
 		mightbespace,
 		alt((
-			map(analyze, Statement::Analyze),
-			map(begin, Statement::Begin),
-			map(cancel, Statement::Cancel),
-			map(commit, Statement::Commit),
-			map(create, Statement::Create),
-			map(define, Statement::Define),
-			map(delete, Statement::Delete),
-			map(ifelse, Statement::Ifelse),
-			map(info, Statement::Info),
-			map(insert, Statement::Insert),
-			map(kill, Statement::Kill),
-			map(live, Statement::Live),
-			map(option, Statement::Option),
-			map(output, Statement::Output),
-			map(relate, Statement::Relate),
-			map(remove, Statement::Remove),
-			map(select, Statement::Select),
-			map(set, Statement::Set),
-			map(show, Statement::Show),
-			map(sleep, Statement::Sleep),
-			alt((map(update, Statement::Update), map(yuse, Statement::Use))),
+			alt((
+				map(analyze, Statement::Analyze),
+				map(begin, Statement::Begin),
+				map(r#break, Statement::Break),
+				map(cancel, Statement::Cancel),
+				map(commit, Statement::Commit),
+				map(r#continue, Statement::Continue),
+				map(create, Statement::Create),
+				map(define, Statement::Define),
+				map(delete, Statement::Delete),
+				map(foreach, Statement::Foreach),
+				map(ifelse, Statement::Ifelse),
+				map(info, Statement::Info),
+				map(insert, Statement::Insert),
+			)),
+			alt((
```

Comment:
Why is this block in a separate alt?

Reply:
Nom alt only supports a certain triple length. Hence it has to be split into multiple triples of multiple parsers each!